matter_server/client/client.py

```diff
@@ -111,13 +111,17 @@ def get_node(self, node_id: int) -> MatterNode:
             return node
         raise NodeNotExists(f"Node {node_id} does not exist or is not yet interviewed")
 
-    async def commission_with_code(self, code: str) -> MatterNodeData:
+    async def commission_with_code(
+        self, code: str, network_only: bool
```

Comment:
Should `network_only` be required parameter or should we set a default like in the server command?

Reply:
Good point, let's make it an optional parameter as well :+1: